src/lib/store.js

```diff
@@ -0,0 +1,89 @@
+var _f     = require('fs-utils'),
+  exists = require('fs').existsSync,
+  extend = require('lodash/object/extend'),
+  path   = require('path');
+
+var write = _f.writeJSONSync,
+  read  = _f.readJSONSync;
+
+var cli = process.AURELIA;
+var _instance;
+
+var Store = (function(){
+
+  function _Store(configPath) {
+    configPath = configPath || process.env.PWD;
+    this.configName = '.aurelia-config.json'
+    this.configPath = configPath +'/'+this.configName;
+    this.configRoot = path.join.bind(path, process.cwd());
+    this.defaultConfig = exists(this.configPath)
```

Comment:
Probably more of a personal preference but obfuscating the value assignment using a ternary operator over a few lines is a bit difficult to read.  Functionally it makes sense.